cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -558,9 +558,12 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
         }
 
         note {
-          p("Note that no specific privileges are required for showing the current user's privileges using either `SHOW USER username PRIVILEGES`, or `SHOW USER PRIVILEGES`.")
+          p("Note that no specific privileges are required for showing the current user's privileges using either `SHOW USER _username_ PRIVILEGES`, or `SHOW USER PRIVILEGES`.")
+        }
+        note {
+          p("Please note that if you are using non native auth providers like LDAP, `SHOW USER PRIVILEGES` will only work in the following limited capacity:" +
+            "It is only possible for a user to get their own privileges. Other users privileges can not be listed when using a non native auth provider.")
```

Comment:
Some minor english-isms, and also change it from "you" to "the user" to be consistant.
```suggestion
        p("Please note that if a non-native auth provider like LDAP is in use, `SHOW USER PRIVILEGES` will only work in the following limited capacity:" +
          "It is only possible for a user to show their own privileges. Other users' privileges cannot be listed when using a non-native auth provider.")
```